presto-lark-sheets/src/main/java/com/facebook/presto/lark/sheets/SheetsConfig.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.lark.sheets;
+
+import com.facebook.airlift.configuration.Config;
+
+public class SheetsConfig
+{
+    public enum Domain
+    {
+        FEISHU, LARK;
+    }
+
+    private Domain appDomain = Domain.FEISHU;
```

Comment:
Just wondering what's the difference between `FEISHU` and `LARK`? This is the lark connector, but the default domain is `FEISHU`?

Reply:
Lark and Feishu are the same product but serving different users. Lark serves global users, while Feishu serves Chinese users. They have different SDK domains.

I have changed the default value to LARK.